.gitignore

```diff
@@ -1,4 +1,5 @@
 _site
 .sass-cache
+build
```

Comment:
I added this because I was using the same git directory for both `master` and `gh-pages`, but maybe I should have been using two entirely separate git directories?

Reply:
I've been using a separate git directory

```
dev/projects/iree       (main repo)
dev/projects/iree-docs  (docs repo)
```

I also have `iree-build` for CMake builds (outside the repo) and `iree-bazel` for just tensorflow/bazel testing, since those builds take up to an hour and I want to stay productive during them :P